pokemongo_bot/walkers/polyline_generator_tester.py

```diff
@@ -3,10 +3,10 @@
 
 import haversine
 import polyline
-
-from polyline_walker import PolylineWalker
-
-a = PolylineWalker('Poststrasse+20,Zug,CH', 'Guggiweg+7,Zug,CH', 100)
+from math import ceil
+from polyline_generator import Polyline
+a = Polyline((47.1706378, 8.5167405), (47.1700271, 8.518072999999998), 100)
+print(a.points)
```

Comment:
This is not called by any code.. I left it for tests Inside there is a piece of code  that explains you how to check at https://developers.google.com/maps/documentation/utilities/polylineutility the points that were generated...